src/codewhisperer/util/supplementalContext/codeParsingUtil.ts

```diff
@@ -68,29 +67,26 @@ export function countSubstringMatches(arr1: string[], arr2: string[]): number {
     return count
 }
 
-export async function isTestFile(editor: vscode.TextEditor, dependencyGraph: DependencyGraph): Promise<boolean> {
-    const languageConfig = utgLanguageConfigs[editor.document.languageId]
-    if (!languageConfig) {
-        // We have enabled the support only for python and Java for this check
-        // as we depend on Regex for this validation.
-        return false
-    }
+export async function isTestFile(
+    filePath: string,
+    languageConfig: { languageId: string; dependencyGraph?: DependencyGraph; fileContent?: string }
```

Comment:
NIT: This makes it easier to know where the value should come from

```suggestion
    languageConfig: { languageId: vscode.TextDocument['languageId']; dependencyGraph?: DependencyGraph; fileContent?: string }
```

Reply:
yea sg, will update